server/server.js

```diff
@@ -0,0 +1,19 @@
+const express = require('express');
+const app = express();
+const path = require('path');
+
+app.use(('/public', express.static(path.join(__dirname, 'public'))));
```

Comment:
you dont need to add the 'public' part in the begging on this  line becuase the folder is already called 'public'. if you add anything in the start of the callback it just changes the folder name when its found online in the client.